cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -1,6 +1,19 @@
-var checkoutConfiguration = window.Configuration;
 var isValid = false;
 var storeDetails;
+var cardNode = document.getElementById('card');
+var oneClickCard = [];
+var card;
+var idealComponent;
+var afterpayComponent;
+var klarnaComponent;
+var maskedCardNumber;
+const MASKED_CC_PREFIX = '************';
+var oneClickValid = false;
+var selectedMethod;
+var componentArr = [];
+
+var checkoutConfiguration = window.Configuration;
+var installments = window.installments.length > 0 ? JSON.parse(window.installments) : null;
```

Comment:
I think it's better to use if statement instead of the conditional operator. Easier to read